modules/fts/pages/fts-monitor.adoc

```diff
@@ -1,11 +1,104 @@
+
+[#fts_fts_monitoring_statistics2]
+image::fts-monitoring-statistics2.png[,820,align=left] 
+
+The statistics display the following graphical charts:
+
+* Search Query Latency: The average time to run the search query and get a response.
+* Search Docs: The number of documents processed during the indexing.
+* Search Disk Size: The total size of disk space occupied by the fts index.
+* Search Disk Files: The number of index files on disk.
+* Search Memory Segments: The number of memory segments in the full text search index across all partitions.
+* Search Disk Segments: The number of disk segments in the full text search index across all resident disk partions.
+* Search Mutations Remaining: The number of mutations not yet indexed. 
+* Search Partitions: The number of actual index partitions.
+* Search Partitions Expected: The number of partitions expected to be scanned for the search query.
+* Search Records to Persist: The number of index records not yet persisted to the disk.
+* Search Index Rate: The rate (In Bytes per second) of indexing plain text.
+* Search Result Rate: The rate (In Bytes per second) of returning the search result.
+* Search Compaction Rate: The rate (In Bytes per second) of compaction of the index segments.
+* Search Query Rate: The number of queries run per second.
+* Search Query Error Rate: The number of queries per second (Including timeouts) that resulted in an error.
+* Search Slow Queries: The number of queries that run slowly (greater than 5 seconds to run) ) per second.
+* Search Query Timeout Rate: The number of queries that timeout per second.
+* Term Searchers Start Rate: The number of term searchers started per second.
+
+== Monitor Runtime Queries
 
 Users of the full text search service might want to monitor the currently running active queries.
 Monitoring helps get insights into slow queries, or for other debugging purposes.
 FTS provides new REST endpoints to supervise the runtime queries.
 
+== API Query Index
+....
+/api/query/index/{indexName}
+....
+
+The users can use the `pass:c[/api/query/index/{indexName}]` endpoint to get the details of all the active queries for any given FTS index in the system. With this endpoint, the users can also use the longerThan argument to filter the queries running beyond the given span of time.
+
+The longerThan duration string is a signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "20s", "-1.5h" or "2h45m".
+Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+ 
+For example, 
+
+[source,shell]
+----
+curl -XGET -H "Content-Type: application/json" -uAdministrator:asdasd 'http://localhost:9200/api/query/index/<indexName>?longerThan=1ms
+----
+
+.Sample Output
+[source,json]
+----
+{ "status": "ok", "totalActiveQueryCount": 3, "filteredActiveQueries": { "indexName": "DemoIndex", "longerThan": "1s", "queryCount": 3, "queryMap": { "4": { "QueryContext": { "query": { "query": "ipa" }, "size": 10, "
```

Comment:
this formating/indentation looks really off. 
The problem is - it isn't readable for the user?
Ideally, this response json has to be JSON formatted.

Reply:
JSON Formatted